### PR TITLE
QA for Auctions with Alessandra

### PR DESCRIPTION
**Pull request by orta**
Attended a Mocktion, found a few issues. Fixed most.

TODOs:

- [x] Register Button doesn't work ( logs seem odd there )
- [x] Remove the profile image, if the auction doesn't have an associated profile
- [x] Tests

293b40cbfae

Things I can live with for now:
- [ ] Refine sometimes clicks through to the artworks

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Error</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td>No CHANGELOG changes made</td>
    </tr>
    
  </tbody>
</table>
  

  
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Warnings</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td>PR is classed as Work in Progress</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td>Needs testing on a Phone if change is non-trivial</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by orta** [comment]
OK @ashfurrow  - this is ready for review

**Comment by orta** [comment]
hrm, danger is still being wrong - boo

**Comment by orta** [comment]
OK, now she's not wrong, but I'm now suspicious

File: Artsy_Tests/View_Tests/ARArtworkActionsViewTests.m [hunk 1]
```diff
@@ -269,19 +269,22 @@ - (void)tappedMoreInfo:(id)sender;
...
     context(@"at auction", ^{
+
         it(@"no bids", ^{
             view.saleArtwork = [SaleArtwork modelWithJSON:@{ @"opening_bid_cents" : @(1000000) }];
+            view.saleArtwork.auction = [Sale modelWithJSON:@{ @"start_at" : @"1976-01-30T15:00:00+00:00", @"end_at" : @"2045-01-30T15:00:00+00:00" }];
             view.artwork = [Artwork modelFromDictionary:@{ @"sold" : @(false) }];
             [view updateUI];
             [view ensureScrollingWithHeight:CGRectGetHeight(view.bounds)];
             [view layoutIfNeeded];
-            expect(view.auctionPriceView).to.haveValidSnapshot();
```
Comment (ashfurrow) [inline Artsy_Tests/View_Tests/ARArtworkActionsViewTests.m]: Did we remove this on purpose?

Comment (orta) [inline Artsy_Tests/View_Tests/ARArtworkActionsViewTests.m]: nope

**Pull request merged**